### PR TITLE
feat(dashboard): add respec-w3c size tracker

### PR DESCRIPTION
**Pull request by sidvishnoi**
Demo: https://respec.org/dashboard/
(Checked in this branch on server, doesn't require server restart as it's in /static)

### CODE REVIEW

File: static/dashboard/size-chart.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  const elem = document.getElementById("size-chart");
+  const chart = new google.visualization.LineChart(elem);
+  chart.setAction({
+    id: "view_commit",
+    text: "View commit ↗️",
+    action() {
+      const { row } = chart.getSelection().slice(-1)[0];
+      const sha = data.getValue(row, 1);
+      window.open(toCommitURL(sha));
```
Comment (marcoscaceres) [inline static/dashboard/size-chart.js:60]:
```suggestion
      window.open(`https://github.com/w3c/respec/commit/${sha}`);
```

File: static/dashboard/size-chart.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+google.charts.setOnLoadCallback(main);
+
+const form = document.getElementById("size-form");
+const columnMap = { time: 0, sha: 1, size: 2, gzipSize: 3 };
+const toCommitURL = sha => `https://github.com/w3c/respec/commit/${sha}`;
+
+async function main() {
+  const entries = await fetchData();
+  const dataTable = getDataTable(entries);
+  drawChart(dataTable);
```
Comment (marcoscaceres) [inline static/dashboard/size-chart.js:11]: nit: do this last? ... like "finally, draw table"

File: static/dashboard/size-chart.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+google.charts.load("current", { packages: ["line", "corechart"] });
+google.charts.setOnLoadCallback(main);
+
+const form = document.getElementById("size-form");
+const columnMap = { time: 0, sha: 1, size: 2, gzipSize: 3 };
+const toCommitURL = sha => `https://github.com/w3c/respec/commit/${sha}`;
```
Comment (sidvishnoi) [inline static/dashboard/size-chart.js:6]:
```suggestion
```

File: static/dashboard/size-chart.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  const text = await fetch("/respec/size").then(res => res.text());
+  const lines = text.trimEnd().split("\n");
+  return lines.map(entry =>
+    JSON.parse(entry, (k, v) => (k === "time" ? new Date(v * 1000) : v)),
+  );
+}
+
+/** @param {google.visualization.DataTable} data */
+function drawChart(data) {
+  const showGzipSize = form.gzipSize.checked;
```
Comment (marcoscaceres) [inline static/dashboard/size-chart.js:25]:
```suggestion
  const { checked: showGzipSize } = form.gzipSize;
```

**Pull request merged**